README.md

```diff
@@ -72,6 +72,49 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-check/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+### What features does `cargo-semver-checks` enable in the tested crates?
+
+By default, crates are built with all features excluding heuristically selected features that usually break semver. More precisely, features named `unstable`, `nightly`, `bench`, `no_std` or starting with `__` are disabled.
+
+The flags below can be used to select the checked features set:
+```
+--default-features
+    Use only the crate-defined default features, as well as any features added explicitly via other flags.
+
+    Using this flag disables the heuristic that enables all features except `unstable`, `nightly`, `bench`, `no_std`, and ones starting with `__`.
+
+--only-explicit-features
+    Use no features except ones explicitly added by other flags.
+
+    Using this flag disables the heuristic that enables all features except `unstable`, `nightly`, `bench`, `no_std`, and ones starting with `__`.
+
+--features <NAME>
+    Add a feature to the set of features being checked. The feature will be used in both the baseline and the current version of the crate
+
+--baseline-features <NAME>
+    Add a feature to the set of features being checked. The feature will be used in the baseline version of the crate only
+
+--current-features <NAME>
+    Add a feature to the set of features being checked. The feature will be used in the current version of the crate only
+
+--all-features
+    Use all the features, including features named `unstable`, `nightly`, `bench`, `no_std` or starting with `__`, that are otherwise disabled by default
+```
+
+Let's say we have a crate with the following features:
+- `default` - the only crate's default feature,
+- `non-default`,
```

Comment:
Let's rename this feature to something else. I know it's `non-default` but the "default" substring there really throws me off in the examples. It's also a possible point of confusion around whether `--default-features` means "features named `default`" or something else.

Consider using real feature names crates would use and users might already be used to. For example, `std` as a default feature, `serde` as an optional feature, etc. That way we don't risk confusing the reader between flags and feature names.

Reply:
I guess it's not exactly what you meant, but `serde` seems like an excellent crate to showcase the functionality.